clang/include/clang/CIR/Dialect/IR/CIRAttrs.h

```diff
@@ -20,6 +20,10 @@
 
 #include "clang/CIR/Interfaces/CIRTypeInterfaces.h"
 
+namespace cir {
+inline constexpr uint32_t DefaultGlobalCtorDtorPriority = 65535;
```

Comment:
I don't think we need this: (a) it's not used in this PR and (b) can be done in tablegen level by assigning a default value for priorities.